src/deb/control/control

```diff
@@ -1,6 +1,6 @@
 Package: [[artifactId]]
 Version: [[version]]
-Description: [[name]]
+Description: [[name]]+[[build.timestamp]]-plos[[build.counter]]
 Architecture: all
```

Comment:
It looks to me like dipro is using the version field: https://github.com/PLOS/content-repo/pull/80/files